python/grass/pygrass/raster/rowio.py

```diff
@@ -18,17 +18,17 @@
 )
```

Comment:
Seems like removing the last `ctypes.c_int` together with removing the `l_what` parameter could work, not tested though.

Reply:
That should be the solution! The first argument to [ctypes.CFUNCTYPE](https://docs.python.org/3/library/ctypes.html#ctypes.CFUNCTYPE) is the `restype`, followed by the argument types of the function (`getmaprow_[CELL|FCELL|DCELL](int, pointer, int)`).

@mshukuno Please try the following change Anna suggested:

```diff
diff --git a/python/grass/pygrass/raster/rowio.py b/python/grass/pygrass/raster/rowio.py
index 377843642a..ed43c9a845 100644
--- a/python/grass/pygrass/raster/rowio.py
+++ b/python/grass/pygrass/raster/rowio.py
@@ -13,9 +13,7 @@ from grass.pygrass.errors import GrassError
 from grass.pygrass.raster.raster_type import TYPE as RTYPE
 
 
-CMPFUNC = ctypes.CFUNCTYPE(
-    ctypes.c_int, ctypes.c_int, ctypes.c_void_p, ctypes.c_int, ctypes.c_int
-)
+CMPFUNC = ctypes.CFUNCTYPE(ctypes.c_int, ctypes.c_int, ctypes.c_void_p, ctypes.c_int)
 
 
 def getmaprow_CELL(fd, buf, row):
```